lib/detect-testing-library-utils.ts

```diff
@@ -145,13 +147,27 @@ export function detectTestingLibraryUtils<
       return !!node.name.match(/^query(All)?By.+$/);
     };
 
+    /**
+     * Determines whether a given node is `findBy*` or `findAllBy*` query variant or not.
+     */
+    const isFindByQuery: DetectionHelpers['isFindByQuery'] = (node) => {
+      return !!node.name.match(/^find(All)?By.+$/);
```

Comment:
This is a tiny bit faster 😅

```suggestion
      return /^find(All)?By.+$/.test(node.name);
```